packages/app/src/specs/flaky-badge/FlakyBadge.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div
+    class="font-bold bg-orange-50 rounded-4px text-xs px-4px text-orange-600 items-center uppercase"
+    style="width: fit-content"
+    data-cy="flaky-badge"
```

Comment:
Two questions for ya!

1. Looks like we have a [Badge](https://github.com/cypress-io/cypress/blob/develop/packages/frontend-shared/src/components/Badge.vue) Vue component in `frontend-shared`; is there a need for `FlakyBadge` as a separate component? 
2. If there is a need for it to be separate, are there any other packages that might use it? In other words, would there be benefit to adding it to the `frontend-shared` package?

Reply:
Was originally going to use the base `Badge`, but the designs call for _slightly_ different colors and font size than is used there. Wasn't sure whether it was better to create a duplicative component, or complicate the base with additional props to support unique styling. Thoughts?

Reply:
Oooooh nice call out! I think it'd be worth reaching out to the design team to get their take, it sounds like it could be an unintended inconsistency in the design. In terms of getting an answer, either @mapsandapps or @elevatebart can bring it up at the Design Review meeting on Tuesday or you can bring it up in the [#design](https://cypressio.slack.com/archives/C3WD9SGKW) channel for a faster response

If it is intended to be different, my personal preference would be to use `Badge` just so that if someone is trying to track down a badge in the code, it's a one-stop-shop. I can see both sides of the argument tho

Reply:
👍 I'll reach out to design to see which direction we should go - I agree it makes the most sense to standardize, but this is a new use case for the component so it may make sense to introduce some variance.

In the short term I've updated `FlakyBadge.vue` to just be a wrapper around `Badge` to handle the necessary style customizations - simplifies it a bit and reduces duplication. When I go to hook this up in the Part 2 PR I'll see whether having this wrapper makes sense based on what design comes back with or whether I should just inline the use of `Badge` directly

Reply:
Adding it to our next design critique 🧐.

It would really be a good idea to have this badge component in the DS.

if [this figma doc](https://www.figma.com/file/1WJ3GVQyMV5e7xVxPg3yID/Design-System?node-id=1950%3A2659) is the "Tag" your are looking for, then I believe [this ClickUp ticket](https://app.clickup.com/t/18033298/DESIGN-655) is what you want to follow.

Reply:
Wanted to throw out we have a Flaky badge on the runs page. Though a different context, I'd expect the styles of the badge and verbiage (All caps vs standard) to be consistent
<img width="294" alt="Screen Shot 2022-07-26 at 11 23 04 AM" src="https://user-images.githubusercontent.com/25158820/181059763-785e11ab-3865-4988-833b-a559fa4eb6bc.png">

Might want to refactor the runs page to use the Badge component.

Reply:
@abelb Can you weigh in on Zach's comment above?

Reply:
@ZachJW34 I'm working with @abelb on figuring out how to standardize this since it impacts the app in two places as well as the dashboard in two places, and we probably want to figure out a design standard for all four places. Will address this in Part 2 of this effort